analysis_options.yaml

```diff
@@ -94,7 +91,6 @@ linter:
     # - parameter_assignments # we do this commonly
     - prefer_adjacent_string_concatenation
     - prefer_asserts_in_initializer_lists
-    - prefer_bool_in_asserts
     - prefer_collection_literals
     - prefer_conditional_assignment
     - prefer_const_constructors
```

Comment:
ideally we'd find a way to make this file always exactly the same as the version in the flutter/flutter repo

Reply:
I imagine a lot of projects would want that functionality -- "start with this canonical analysis options file, then override it with the following options"

Perhaps the analyzer could beef up its `include` directive to support not only relative paths, but also an enumerated set of SDKs?  In this world, we'd move the canonical analysis options that currently live in the flutter repo into the Dart SDK as the "flutter analysis options", then the flutter repo would just say "I use Flutter's options"...

/cc @pq

Reply:
Somewhat like clang-format use chromium style?

Reply:
I would like that too - I usually start a new Flutter project by copying the Flutter repo's analysis_options

Reply:
https://github.com/flutter/flutter/pull/26244

Reply:
That has landed - pulling it in to this will require a bunch more changes in the implementation files (get rid of optional new, use = instead of `:` for parameters, etc.) that would be better to do in a separate PR.

Reply:
sgtm